pkg/aspect/config/accessLogs.proto

```diff
@@ -21,54 +21,63 @@ option go_package="config";
 // Configures an individual access-logs aspect.
 //
 // Example usage:
-// kind: access-logs
-// params:
-//    log_name: "logs/access_log"
-//    attributes: "source.ip","host.URL","
-//    log_format: "COMMON"
+//     kind: access-logs
+//     params:
+//        log_name: "logs/access_log"
+//        log:
+//            log_format: CUSTOM
+//            descriptor_name: "MyCustomDescriptor"
+//            labels:
+//                time: response.time
+//                user_agent: request.user-agent | "Mozilla/5.0"
 message AccessLogsParams {
-    // Provides a way to identify a collection of related log entries.
-    // Default value: "access_log"
+    // Identifies a collection of related log entries.
     string log_name = 1;
 
-    // A list of names of attributes that will be used to generate
-    // access log entries.
-    repeated string attributes = 2;
+    // Describes how attributes must be evaluated to produce values for a log message.
+    message AccessLog {
+        // AccessLogFormat specifies the format to use for generating textual log
+        // entries for the accessLogger aspect of the mixer.
+        //
+        // Default value: COMMON.
```

Comment:
Default value is no longer COMMON, now that we have added ACCESS_LOG_FORMAT_UNSPECIFIED.

We should update documentation.